InformationScripting/src/graph/Property.cpp

```diff
@@ -33,5 +33,10 @@ boost::python::object pythonObject(const Property& p)
 	return p.data_->pythonObject();
 }
 
+bool Property::operator==(const Property& other) const
```

Comment:
I wonder if this would enable us to simply ask the question: `InofrmationNodeA == InformationNodeB`? If so, then this means that two nodes are equal if *all* of their data is equal, but I can imagine cases where we want a less strict definition of equality, as data might have been attached/altered in one node and not in the other, even though on some logical level they represent the same entity.